packages/babel-preset-app/src/index.js

```diff
@@ -32,15 +32,20 @@ function getDefaultPolyfills (corejs) {
 }
 
 function getPolyfills (targets, includes, { ignoreBrowserslistConfig, configPath, corejs }) {
-  const { isPluginRequired } = require('@babel/preset-env')
+  const { default: getTargets, isRequired } = require('@babel/helper-compilation-targets')
```

Comment:
Should we add direct `@babel/helper-compilation-targets` dependency?

Reply:
I think it's find, `@babel/helper-compilation-targets` is also used by `@babel/preset-env` and other packages

Reply:
As we directly require this module I think we need to define in package.json. At least it is a requirement of yarn v2 and similar new PMs.

Reply:
I'll add it.

Reply:
@pi0 Thanks